### PR TITLE
feat: implement --sort-by parameter

### PR DESCRIPTION
**Pull request by marekaf**

### CODE REVIEW

File: internal/worker/sort.go [hunk 1]
```diff
@@ -24,10 +24,29 @@ func (a ByTimeCreated) Less(i, j int) bool {
...
+	if err != nil {
+		log.Fatal(err)
+	}
 
+	jnum, err := strconv.Atoi(a[j].TimeUploadedMs)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return inum < jnum
```
Comment (yteraoka) [inline internal/worker/sort.go]: To keep newer tag, I think this should be

```
return inum > jnum
```

**Comment by yteraoka** [comment]
I found the `TimeCreatedMs` in `olderThanRetention()`.

https://github.com/marekaf/gcr-lifecycle-policy/blob/870f3d43f37f5126cf42875bf16896661babf669/internal/worker/cleanup.go#L73-L83

File: internal/worker/sort.go (hunk 1 above)
Comment (marekaf) [reply internal/worker/sort.go]: You sure? I'll write some unit tests just to be sure :))

Comment (marekaf) [reply internal/worker/sort.go]: I think you are right! I fixed it and wrote some tests. Thanks for the help!

**Pull request merged**